src/main/java/core/basesyntax/reader/FileReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.reader;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
```

Comment:
The class implementation should not start with an empty line. Please remove the empty line before the class declaration.